### ISSUE TITLE
visualization.visualize_image_attr crashes with matplotlib 3.7

### ISSUE THREAD

**Issue by daniel-ziegler** [pytorch/captum#1114]
## To Reproduce

1. Install matplotlib 3.7.0
2. Make any call to `visualization.visualize_image_attr`

```
  File "/home/dmz/miniconda3/envs/audit_vis/lib/python3.10/site-packages/captum/attr/_utils/visualization.py", line 250, in visualize_image_attr
    plt_axis.grid(b=False)
  File "/home/dmz/miniconda3/envs/audit_vis/lib/python3.10/site-packages/matplotlib/axes/_base.py", line 3196, in grid
    self.xaxis.grid(visible, which=which, **kwargs)
  File "/home/dmz/miniconda3/envs/audit_vis/lib/python3.10/site-packages/matplotlib/axis.py", line 1655, in grid
    self.set_tick_params(which='major', **gridkw)
  File "/home/dmz/miniconda3/envs/audit_vis/lib/python3.10/site-packages/matplotlib/axis.py", line 927, in set_tick_params
    kwtrans = self._translate_tick_params(kwargs)
  File "/home/dmz/miniconda3/envs/audit_vis/lib/python3.10/site-packages/matplotlib/axis.py", line 1071, in _translate_tick_params
    raise ValueError(
ValueError: keyword grid_b is not recognized; valid keywords are ['size', 'width', 'color', 'tickdir', 'pad', 'labelsize', 'labelcolor', 'zorder', 'gridOn', 'tick1On', 'tick2On', 'label1On', 'label2On', 'length', 'direction', 'left', 'bottom', 'right', 'top', 'labelleft', 'labelbottom', 'labelright', 'labeltop', 'labelrotation', 'grid_agg_filter', 'grid_alpha', 'grid_animated', 'grid_antialiased', 'grid_clip_box', 'grid_clip_on', 'grid_clip_path', 'grid_color', 'grid_dash_capstyle', 'grid_dash_joinstyle', 'grid_dashes', 'grid_data', 'grid_drawstyle', 'grid_figure', 'grid_fillstyle', 'grid_gapcolor', 'grid_gid', 'grid_in_layout', 'grid_label', 'grid_linestyle', 'grid_linewidth', 'grid_marker', 'grid_markeredgecolor', 'grid_markeredgewidth', 'grid_markerfacecolor', 'grid_markerfacecoloralt', 'grid_markersize', 'grid_markevery', 'grid_mouseover', 'grid_path_effects', 'grid_picker', 'grid_pickradius', 'grid_rasterized', 'grid_sketch_params', 'grid_snap', 'grid_solid_capstyle', 'grid_solid_joinstyle', 'grid_transform', 'grid_url', 'grid_visible', 'grid_xdata', 'grid_ydata', 'grid_zorder', 'grid_aa', 'grid_c', 'grid_ds', 'grid_ls', 'grid_lw', 'grid_mec', 'grid_mew', 'grid_mfc', 'grid_mfcalt', 'grid_ms']
```

## Expected behavior

It should not give an error. To be compatible with newer versions of matplotlib, Captum should call `plt_axis.grid(visible=False)` instead of passing `b=False`.

## Environment
Describe the environment used for Captum

```

 - Captum / PyTorch Version (e.g., 1.0 / 0.4.0): Pytorch 1.11.0, Captum 0.6.0
 - OS: Linux
 - How you installed Captum / PyTorch (`conda`, `pip`, source): pip
 - Python version: 3.10.0
 - CUDA/cuDNN version: 12.0
 - GPU models and configuration: A100

**Comment by vivekmig** [comment]
Thanks for reporting the issue @daniel-ziegler ! We will push a fix for this soon

**Comment by MJGroeneveld** [comment]
Is this already fixed? Because I get the same error with matplotlib version 3.7.1

**Comment by tendermonster** [comment]
The "captum/attr/_utils/visualisation.py" should be updated by someone to work correctly

Removing this line in captum/attr/_utils/visualisation.py solves the problem. 
```
# Remove ticks and tick labels from plot.
# visualization.visualize_image_attr line 250 or so
plt_axis.xaxis.set_ticks_position("none")
plt_axis.yaxis.set_ticks_position("none")
plt_axis.set_yticklabels([])
plt_axis.set_xticklabels([])
print("hi i am the error")
plt_axis.grid(b=False) # <- this line
```

**Comment by strangemonad** [comment]
@vivekmig Any luck with this? is this a good issue to tag with help-wanted?

**Comment by tendermonster** [comment]
I mean, the simplest thing to do here is to go from
plt_axis.grid(b=False)
to 
plt_axis.grid() or plt_axis.grid(visible=False) or something

**Comment by vivekmig** [comment]
Apologies for the delay, this issue was fixed in #1118 to use the new visible parameter, and this fix will be available in the next release. Thanks for reporting this issue!
